airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/source.py

```diff
@@ -88,7 +63,6 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         Show error message in case of request exception or unexpected response.
         """
         try:
-            config = ConnectorConfig.parse_obj(config)  # FIXME: this will be not need after we fix CDK
```

Comment:
Actually, we wanted to make pydantic config as a default in future CDK releases. So what's the point of getting rid of it?

Reply:
The point is to not do the same twice: in spec.json and here